taxonomies/categories.txt

```diff
@@ -32832,7 +32864,7 @@ ciqual_food_name:fr:Yaourt, lait fermenté ou spécialité laitière, nature
 
 <en:Yogurts
 en:Plain fermented milk yogurt with cream, Plain dairy specialty with cream
```

Comment:
When Ciqual has a category name like "ciqual_food_name:en:Yogurt, fermented milk or dairy specialty, plain, w cream"

our name should be "Plain yogurts with cream, plain fermented milks with cream, plain dairy specialties with cream".

Same for all yogurts entries. "Yaourts au lait fermenté" should just be "Yaourts, Laits fermentés".

Reply:
Fixed in the next commit